tests/test_filtering.py

```diff
@@ -1,9 +1,111 @@
 from kbmod import *
+from analysis_utils import PostProcess
 import numpy as np
 import tempfile
 import unittest
 
-class test_kernels_wrappers(unittest.TestCase):        
+class test_kernels_wrappers(unittest.TestCase):
+    def setUp(self):
+        # Set up initial variables for analysis_utils.PostProcess
+        self.config = {}
+        self.config['num_cores'] = 1
+        self.config['sigmaG_lims'] = [0., 1.]
+        self.config['eps'] = None
+        self.config['cluster_type'] = None
+        self.config['cluster_function'] = None
+        self.config['clip_negative'] = None
+        self.config['mask_bits_dict'] = None
+        self.config['flag_keys'] = None
+        self.config['repeated_flag_keys'] = None
+        
+        # Set up old_results object for analysis_utils.PostProcess
+        psi_curves = [[1., 1., 1.], [1., 1., 1.], [1., 1., 1.]]
+        phi_curves = [[1., 1., 1.], [1., 1., 1.], [1., 1., 1.]]
+        results = [1., 1., 1.]
+        
+        self.old_results = {}
+        self.old_results['psi_curves'] = psi_curves
+        self.old_results['phi_curves'] = phi_curves
+        self.old_results['results'] = results
+        
+        # test pass thresholds
+        self.pixel_error = 0
+        self.velocity_error = 0.05
+        self.flux_error = 0.15
+
+        # image properties
+        self.imCount = 3
+        self.dim_x = 80
+        self.dim_y = 60
+        self.noise_level = 8.0
+        self.variance = self.noise_level**2
+        self.p = psf(1.0)
+
+        # object properties
+        self.object_flux = 250.0
+        self.start_x = 17
+        self.start_y = 12
+        self.x_vel = 21.0
+        self.y_vel = 16.0
+
+        # create a trajectory for the object
+        self.trj = trajectory()
+        self.trj.x = self.start_x
+        self.trj.y = self.start_y
+        self.trj.x_v = self.x_vel
+        self.trj.y_v = self.y_vel
+
+        # search parameters
+        self.angle_steps = 150
+        self.velocity_steps = 150
+        self.min_angle = 0.0
+        self.max_angle = 1.5
+        self.min_vel = 5.0
+        self.max_vel = 40.0
+
+        # Select one pixel to mask in every other image.
+        self.masked_x = 5
+        self.masked_y = 6
+        
+        self.imlist = []
+        for i in range(self.imCount):
+            time = i/self.imCount
+            im = layered_image(str(i), self.dim_x, self.dim_y, 
+                                self.noise_level, self.variance, time,
+                                self.p)
+            im.add_object(self.start_x + time*self.x_vel+0.5,
+                           self.start_y + time*self.y_vel+0.5,
+                           self.object_flux)
```

Comment:
Small comment: There is an extra space at the start of lines 77 and 78. They should be aligned with self.start_x.